CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-### 0.4.5 (Next)
+### 1.0.0 (Next)
```

Comment:
Also bump it in https://github.com/dblock/iex-ruby-client/blob/master/lib/iex/version.rb